README.md

```diff
@@ -47,6 +47,18 @@ in pkgs.mkShell {
 
 After this, enter into your nix-shell and you are good to go.
 
+## Updating
+
+### To update the whole environment, do the following in order:
+
+- `niv init` will update nix/sources.nix if a new one is available
+- backup your nix/sources.json: `cp nix/sources.json nix/sources.json.bak`
```

Comment:
I think this step is not necessary because in this case, we are using `git`.
Maybe add it below as a notice to the `niv update` command?
Like:

```
  - if you are not using a VCS (like git), don't forget to backup your `nix/sources.json` file